spec/features/email_spec.rb

```diff
@@ -29,15 +29,16 @@
 
   scenario "logged in as Organizer after sending an Email to the applicants the event application status gets locked" do
     login(:organizer)
-    @event.application_status_locked = false
+    @event.acceptances_have_been_sent = false
```

Comment:
Would it be nicer to have an event in selection phase than setting those values by hand? You might have to check that participant_selection_locked is false at the beginning, tho